package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -641,8 +641,13 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
          * @param {Object} value
          */
         parameterChanged: function (param, name, value) {
-          if (this.onParameterChanged) {
+          if (this.onParameterChanged && typeof this.onParameterChanged === 'function') {
             this.onParameterChanged(name, value);
+          } else {
+            var paramCallback = this.onParameterChanged[name];
```

Comment:
We can have an exception in this line for situation: api.event.parameterChanged(null);